tests/conftest.py

```diff
@@ -71,6 +111,8 @@ def dbsession(engine, dbsession, sqlalchemy_keep_db) -> Session:
     finally:
         # Drop all test tables after the tests completed
         if not sqlalchemy_keep_db:
+            from schematools.importer.base import metadata
+
             metadata.drop_all(bind=engine)
```

Comment:
This `metadata` is undefined if `sqlalchemy_keep_db` is False.

Reply:
Good catch. Left over from an experiment that wasn't needed after all. Fixing.